examples/static_react_task_with_tips/README.md

```diff
@@ -0,0 +1,131 @@
+# Simple Static React Task With Tips
```

Comment:
Having a copy-pasted duplicated README here creates "two sources of truth" that we have to maintain on an on-going basis. I would much prefer linking to the other README from this README, and only keeping the additional "delta" information on Tips usage here.

Reply:
Agreed